### ISSUE TITLE
Fail to load large Tif file

### ISSUE THREAD

**Issue by phlavenk** [JuliaIO/FileIO.jl#261]
I've problem loading following "correct" tiff file (had to pack it to zip to upload):

[Bond_Pad.zip](https://github.com/JuliaIO/FileIO.jl/files/4680197/Bond_Pad.zip)

All the others tiffs I can load correctly, this one is bigger 2048x1364 (8-bit) and fails with:
```julia
Error encountered while loading "c:\\Users\\petr.hlavenka\\.julia\\dev\\PhaseViewer\\data\\exp_raw\\Bond_Pad.tif".

Fatal error:
ERROR: UnableToOpenBlob `c:\Users\petr.hlavenka\.julia\dev\PhaseViewer\data\exp_raw\Bond_Pad.tif': No such file or directory @ error/blob.c/OpenBlob/2873
Stacktrace:
 [1] handle_error(::ErrorException, ::FileIO.File{FileIO.DataFormat{:TIFF}}) at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\error_handling.jl:82
 [2] handle_exceptions(::Array{Any,1}, ::String) at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\error_handling.jl:77
 [3] load(::FileIO.Formatted; options::Base.Iterators.Pairs{Union{},Union{},Tuple{},NamedTuple{(),Tuple{}}}) at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\loadsave.jl:186
 [4] load at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\loadsave.jl:166 [inlined]
 [5] #load#13 at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\loadsave.jl:118 [inlined]
 [6] load(::String) at C:\Users\petr.hlavenka\.julia\packages\FileIO\ZknoK\src\loadsave.jl:118
 [7] top-level scope at REPL[6]:1
 [8] eval(::Module, ::Any) at .\boot.jl:331
 [9] eval_user_input(::Any, ::REPL.REPLBackend) at D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.4\REPL\src\REPL.jl:86
 [10] run_backend(::REPL.REPLBackend) at C:\Users\petr.hlavenka\.julia\packages\Revise\Pcs5V\src\Revise.jl:1073
 [11] top-level scope at none:0
```

**Comment by SimonDanisch** [comment]
```
UnableToOpenBlob `c:\Users\petr.hlavenka\.julia\dev\PhaseViewer\data\exp_raw\Bond_Pad.tif': No such file or directory @ error/blob.c/OpenBlob/2873
```
Any chance you messed up the path in a subtle way?

**Comment by phlavenk** [comment]
Definitely not. If I on purpose alter the path, the error is different.

**Comment by SimonDanisch** [comment]
What's the error in that case?

> No such file or directory

That seems pretty clear :D Maybe a windows path problem?

**Comment by SimonDanisch** [comment]
I can open it on linux:
```julia
julia> using FileIO, ImageMagick
[ Info: Precompiling ImageMagick [6218d12a-5da1-5696-b52f-db25d2ecc6d1]

julia> load("Bond_Pad.tif")
1364×2048 Array{Gray{N0f8},2} with eltype ColorTypes.Gray{FixedPointNumbers.Normed{UInt8,8}}:
```

**Comment by johnnychen94** [comment]
> I can open it on linux

It also works in MacOS.

**Comment by timholy** [comment]
It would be worth checking whether TiffImages fixes this, see #290

**Comment by johnnychen94** [comment]
> It would be worth checking whether TiffImages fixes this, see #290

```julia
julia> using TiffImages

julia> TiffImages.load("/Users/jc/Downloads/Bond_Pad.tif")
1364×2048 TiffImages.DenseTaggedImage{ColorTypes.Gray{FixedPointNumbers.N0f8}, 2, UInt32, Matrix{ColorTypes.Gray{FixedPointNumbers.N0f8}}}:
```
